kroxylicious-runtime/src/test/java/io/kroxylicious/proxy/config/ConfigParserTest.java

```diff
@@ -268,6 +271,67 @@ void shouldDetectDuplicateClusterNodeNames() {
 
     }
 
+    @Test
+    void systemPropertyExpansion() {
+
+        System.setProperty("bootstrap", "zzzz");
+        System.setProperty("port", "1234");
```

Comment:
Junit Pinoeer has an extension for managing [system properties](https://junit-pioneer.org/docs/system-properties/) so tests can run in a properly isolated environment.